kroxylicious-filters/kroxylicious-encryption/src/test/java/io/kroxylicious/filter/encryption/inband/InBandKeyManagerTest.java

```diff
@@ -346,34 +305,6 @@ void afterWeFailToLoadADekTheNextEncryptionAttemptCanSucceed() {
         assertThat(encrypted).hasSize(2);
     }
 
-    @Test
-    void decryptionFailure() {
```

Comment:
found this was duplicated by `edekDecryptionRetryFailurePropagatedToDecryptCompletionStage`

Reply:
and also was testing logic moved to the ResilientKms and is no longer the InBandKeyManager's concern